ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/MonoGame.Application.WindowsDX.CSharp/MGNamespace.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
+    <MonoGamePlatform>Windows</MonoGamePlatform>

```

Comment:
`MonoGamePlatform` does not necessarily have to be set in the templates, because the MonoGame.Framework package comes with a props file that sets it for you. It might be nice to quickly set up a project with a P2P reference for source debugging though, and it doesn't do any harm having this line so let's leave it in.